www/src/pages/foundations/colors.jsx

```diff
@@ -106,8 +107,8 @@ export default function ({ data }) {
         // $color_name-level <br/>
         $primary-100<br/>
         $primary-200<br/>
-        $primary-300<br/>
-        $primary-400
+        $brand-100<br/>
```

Comment:
Was `primary-300` and `primary-400` the colors for the secondary button option?

Reply:
No I just changed it here to have more varying options.